apps/heft/src/plugins/CopyStaticAssetsPlugin.ts

```diff
@@ -56,19 +107,41 @@ export class CopyStaticAssetsPlugin extends CopyFilesPlugin {
       heftConfiguration.rigConfig
     );
 
-    const destinationFolders: string[] = ['lib'];
+    const destinationFolders: Set<string> = new Set<string>();
+
+    const tsconfigDestinationFolder: string | undefined = await this._tryGetTsconfigOutDirAsync(
+      heftConfiguration.buildFolder,
+      terminal
+    );
+    if (tsconfigDestinationFolder) {
+      destinationFolders.add(tsconfigDestinationFolder);
+    }
+
     for (const emitModule of typescriptConfiguration?.additionalModuleKindsToEmit || []) {
-      destinationFolders.push(emitModule.outFolderName);
+      destinationFolders.add(emitModule.outFolderName);
     }
 
     return {
       ...typescriptConfiguration?.staticAssetsToCopy,
 
       // For now - these may need to be revised later
       sourceFolder: 'src',
```

Comment:
Is this also something that can be extracted from the tsconfig?

Reply:
We may be able to extract it from `rootDir`, but I'd want to do that separately in case that causes a problem.